doc/ledger3.texi

```diff
@@ -6430,6 +6433,9 @@ Aliases are completely ignored.
 @item --no-color
 Suppress any color TTY output.
 
+@item --no-pager
+Direct output to stdout, avoiding pager program.
+
```

Comment:
Thanks for pointing out my limited view on the documentation, I really hadn't thought about the PDF output, and will change my mindset accordingly.

What would be possible is to use `@ifhtml . . . @end ifhtml` for example to link to the balance assertions in the documentation of the `--permissive` option when producing HTML output in addition your `@findex` change.

Another thing I just noticed while going through the [Texinfo reference card](http://svn.savannah.gnu.org/viewvc/*checkout*/trunk/doc/refcard/txirefcard-a4.pdf?root=texinfo&content-type=text%2Fplain): there are other index commands, e.g. `@vindex` for variables. Unfortunately I did not find one for options, but I think it would be nice to separate some of the index entries. What are your thoughts on this?